### PR TITLE
Put the list of kafka connectors on the kafka connect landing page

### LINKED ISSUE #1273: Fix Kafka Connect list-of-connector page to point to existing devportal articles
# What's wrong?

(be specific, screenshots or a copy/paste of the affected part of the page is useful)
[Kafka Connect list-of-connector page](https://docs.aiven.io/docs/products/kafka/kafka-connect/concepts/list-of-connector-plugins.html) contains a lot of links to external repos/docs while the documentation has been written in devportal. We should therefore move to links to internal pages

# URL of affected page, and any other information

### PR DESCRIPTION
**Pull request by lornajane**
# What changed, and why it matters

Definitely a work in progress, to maybe-possibly group the connectors and make a bit more of a landing page feature. 
Fix #1273

### CODE REVIEW

**Comment by ftisiot** [comment]
I like the idea! I would probably use the cards if we can have multiple links in them to have a more compacted view (something like the image below)

home page with cards

**Comment by ftisiot** [comment]
I believe we can move away from the current list of connectors once #1377 is merged

**Comment by lornajane** [comment]
Do you mean the cards should be in two columns like the home page? Or one card per connector?

**Comment by ftisiot** [comment]
I would use the same 2 column approach

**Comment by lornajane** [comment]
Next question: we're linking out to the docs, but we also have some resources for each of the connectors (or most of them). Should we reconsider where we link to, and show them the Aiven examples?

**Comment by ftisiot** [comment]
Yep, now that #1377 has been merged, we can change the links... let me work on this later today

**Comment by ftisiot** [comment]
I moved to internal links, and added the redirect. Now we are missing:

- What to do with the `Requesting new connectors` section present in the original `list-of-connector-plugins`
- Remove the `list-of-connector-plugins` page

**Comment by ftisiot** [comment]
Ready to be reviewed!

File: docs/products/kafka/kafka-connect/howto/request-new-connector.rst [hunk 1]
```diff
@@ -0,0 +1,14 @@
+Request a new connector
+=======================
+
+If you know about new and interesting Apache Kafka®ive connectors you'd like us to support, please send us an email to support@Aiven.io about it to help us shaping the future roadmap. We will evaluate the requested connector and might add support for it.
```
Comment (harshini-rangaswamy) [inline docs/products/kafka/kafka-connect/howto/request-new-connector.rst:4]: This "interesting Apache Kafka®ive connectors" should be "interesting Apache Kafka® connectors"?

Comment (harshini-rangaswamy) [inline docs/products/kafka/kafka-connect/howto/request-new-connector.rst:4]:
```suggestion
If you know about new and interesting Apache Kafka® connectors you'd like us to support, please email us at support@Aiven.io to help us shape the future roadmap. We will evaluate the requested connector and might add support for it.
```

File: docs/products/kafka/kafka-connect/howto/request-new-connector.rst [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+Aiven evaluation process for new Apache Kafka Connect connectors checks:
```
Comment (harshini-rangaswamy) [inline docs/products/kafka/kafka-connect/howto/request-new-connector.rst:6]:
```suggestion
Aiven's evaluation process for new Apache Kafka Connect connectors checks:
```

**Review by harshini-rangaswamy — CHANGES_REQUESTED** [review]
Some minor changes to the "Request a new connector" topic. The rest looks good.

**Review by harshini-rangaswamy — APPROVED** [review]
Looks good!

**Pull request merged**